python/lsst/meas/deblender/deblend.py

```diff
@@ -163,8 +163,8 @@ def addSchemaKeys(self, schema):
                                          doc='Number of children this object has (defaults to 0)')
         self.psfKey = schema.addField('deblend_deblendedAsPsf', type='Flag',
                                       doc='Deblender thought this source looked like a PSF')
-        self.psfCenterKey = schema.addField('deblend_psfCenter', type='PointD',
-                                         doc='If deblended-as-psf, the PSF centroid')
+        self.psfCenterKey = afwTable.Point2DKey.addFields(schema, 'deblend_psfCenter',
+                                         'If deblended-as-psf, the PSF centroid', "pixels")
         self.psfFluxKey = schema.addField('deblend_psfFlux', type='D',
```

Comment:
I'm afraid there aren't any keyword arguments for these.  The Schema addFields methods that did were a special case where I put a lot of effort into making them available, and I'm not going to do that for all the FunctorKeys until we take the decision (and decide on an approach) to support keyword arguments more broadly in the stack.